pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -660,7 +660,7 @@ func (os *OpenStack) getVolume(diskName string) (volumes.Volume, error) {
 }
 
 // Create a volume of given size (in GiB)
-func (os *OpenStack) CreateVolume(name string, size int, tags *map[string]string) (volumeName string, err error) {
+func (os *OpenStack) CreateVolume(name string, size int, opt *map[string]string, tags *map[string]string) (volumeName string, err error) {
```

Comment:
Please use named parameters (or `struct`) instead of a generic map. See gce provisioner for an example:

```
func (gce *GCECloud) CreateDisk(name string, diskType string, zone string, sizeGb int64, tags map[string]string) error
```